packages/salesforcedx-vscode-core/src/decorators/scratch-org-decorator.ts

```diff
@@ -5,20 +5,12 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { projectPaths } from '@salesforce/salesforcedx-utils-vscode';
 import * as fs from 'fs';
-import * as os from 'os';
-import * as path from 'path';
 import { StatusBarAlignment, StatusBarItem, window, workspace } from 'vscode';
 import { nls } from '../messages';
-import { workspaceUtils } from '../util';
 
-const CONFIG_FILE = workspaceUtils.hasRootWorkspace()
-  ? path.join(
-      workspaceUtils.getRootWorkspacePath(),
-      '.sfdx',
-      'sfdx-config.json'
-    )
-  : path.join(os.homedir(), '.sfdx', 'sfdx-config.json');
```

Comment:
hummmmmmmmmmm. I gotta look at the source more closely here.  I don't really understand how you could want the global config file in this flow.